src/spells.cpp

```diff
@@ -1137,9 +1137,9 @@ bool RuneSpell::executeUse(Player* player, Item* item, const Position&, Thing* t
 
 	if (needTarget) {
 		if (!target) {
-			Tile* toTile = g_game.map.getTile(toPosition);
+			auto toTile = g_game.map.getTile(toPosition);
```

Comment:
Could this be `const auto`
```suggestion
			const auto toTile = g_game.map.getTile(toPosition);
```